.github/workflows/draft.yml

```diff
@@ -34,21 +35,20 @@ jobs:
           netlify deploy --dir build --message '${{ github.event.commits[0].message }}' > log.tmp.txt 2>&1
           cat log.tmp.txt | grep -E 'Logs|Website Draft URL' > log.txt
       - name: Read deployment log
+        if: '! github.event.pull_request.head.repo.fork'
```

Comment:
Let's make them dependent on eachother using `needs:` like we do in the `core` CI.

Reply:
You mean a `needs` in a step? I don't think that is possible

Reply:
Oh wait, these are not separate steps. Nevermind then. I was just thinking of a way to only do the logic for the `if` in one place so we don't forget the others when we add extra conditions

Reply:
Completely agreed but I couldn't find a less ugly way :)